src/front/glsl/parser_tests.rs

```diff
@@ -305,19 +394,23 @@ fn functions() {
     )
     .unwrap();
 
-    parse_program(
+    parse(
+        &mut parser,
```

Comment:
can we make `parse` as a method on the parser, for convenience?

Reply:
Sorry I don't quite understand what you're proposing, can you elaborate?

Reply:
Okay I  think I've figured out what you meant.